src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultProjectSnapshotProjectEngineFactory.cs

```diff
@@ -71,7 +71,7 @@ public override IProjectEngineFactory FindFactory(IProjectSnapshot project)
             throw new ArgumentNullException(nameof(project));
         }
 
-        return SelectFactory(project.Configuration ?? s_defaultConfiguration, requireSerializable: false);
```

Comment:
Can `s_defaultConfiguration` be removed? Looks like the one place its used is redundant anyway.

(also could totally be done as a follow up to null-enable this whole class)

Reply:
Yup. This is coming in my larger MEF PR, and I didn't want to pull the larger refactoring of this class into this PR.